aod-discord-bot.js

```diff
@@ -4067,7 +4067,8 @@ global.setRolesForMember = setRolesForMember;
 
 //guildMemberAdd event handler -- triggered when a user joins the guild
 client.on('guildMemberAdd', member => {
-	setRolesForMember(member.guild, member, 'First time join');
+	setRolesForMember(member.guild, member, 'First time join')
+		.catch(console.error);
```

Comment:
I added a statement to catch a rejected promise if the database is unavailable rather than terminating the bot. This helps for local development, but I can pull it out if it will affect the live bots.

Reply:
I don't have the systemd service setup to catch stderr. Can you alter to console.log please?

This should not cause any issues and is probably the right thing to do.

Reply:
Fixed in 513e764.